torch/_dynamo/variables/torch.py

```diff
@@ -365,6 +373,22 @@ def get_state_from_generator():
                 ),
                 **options,
             )
+        if (
+            any([isinstance(x, DynamicShapeVariable) for x in args])
```

Comment:
Following torch.numel above, the test here can be tight for 1 arg, since math.sqrt only takes one arg. What I wouldn't give to be allowed to use pattern matching lol